src/formula.rs

```diff
@@ -325,6 +360,58 @@ impl PartialEq for Formula {
     }
 }
 
+impl<'a> IntoIterator for &'a Formula {
+    type Item = (Label, bool);
+    type IntoIter = LabelIter<'a>;
+
+    #[inline]
+    fn into_iter(self) -> Self::IntoIter {
+        LabelIter {
+            formula: self,
+            stack: vec![(self.root, self.tree.children_iter(self.root))],
+        }
+    }
+}
+
+/// An iterator through the labels of a formula.
+/// This iterator sequence is depth-first, postfix (post-order).
```

Comment:
Postorder traversal is surprising. This might be better expressed as an explicit function on Formula like `labels_postorder()` instead of just `iter()/into_iter()`.

Reply:
If that is surprising, I will change it into a pre-order iterator and rename it just `labels`.